pkgs/development/python-modules/tld/default.nix

```diff
@@ -26,6 +26,10 @@ buildPythonPackage rec {
     tox
   ];
 
+  preCheck = ''
+    # these tests require network access, but disabledTestPaths doesn't work
+    echo > src/tld/tests/test_commands.py
```

Comment:
I would rather rm the file, as that's more explicit to what you're doing.

Reply:
I tried that, but another file (an `__init__.py`) tries to import it, and fails if you simply delete it.

Reply:
can you add that as a comment? :)

Reply:
Done